docs/user/cli-reference.md

```
@@ -1447,6 +1447,52 @@ aicr trust update
 
 ---
 
+### aicr skill
+
+Generate an AI agent skill file that teaches a coding agent how to use the AICR CLI. The generated file is written to the agent's standard configuration directory.
+
+**Synopsis:**
+```shell
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)